blueprints/smart/files/__root__/__smart__/__name__.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+
+const propTypes = {
+}
+
+class <%= pascalEntityName %> extends Component {
+  render() {
+    return (
+      <div></div>
+    );
+  }
+}
+
+const mapStateToProps = (state) => {
```

Comment:
It is more convenient to use short forms of the map functions. Also, these should be placed at the top the file. This way you can see data flow before component code that uses this data.